docs/examples/pydata.md

```diff
@@ -95,25 +93,10 @@ data
 
 `ipyleaflet` is a **Jupyter**/**Leaflet** bridge enabling interactive maps in the Jupyter notebook environment. this demonstrate how you can integrate maps in your documentation.
 
-```{code-cell}
+```{jupyter-execute}
```

Comment:
don't we want this to be `code-cell` now, so that we don't have the extra build cost of running the jupyter-execute kernel again?

Or if we _need_ to use `jupyter-execute` for this example, maybe we could move it to the `execution.rst` page?

Reply:
I think it's legacy because I started before you moved the pydata page to an executed one